internal/cloud/realm/dependencies.go

```diff
@@ -49,20 +82,43 @@ func (c *client) ImportDependencies(groupID, appID, uploadPath string) error {
 		return err
 	}
 
-	res, resErr := c.do(
-		http.MethodPost,
+	res, err := c.do(
+		http.MethodPut,
 		fmt.Sprintf(dependenciesPathPattern, groupID, appID),
 		api.RequestOptions{
 			Body:        body,
 			ContentType: w.FormDataContentType(),
 		},
 	)
-	if resErr != nil {
-		return resErr
+	if err != nil {
+		return err
 	}
 	if res.StatusCode != http.StatusNoContent {
-		return api.ErrUnexpectedStatusCode{"import dependencies", res.StatusCode}
+		return fmt.Errorf("import got invalid status code %d", res.StatusCode)
+	}
+
+	waitForTranspilation := func() error {
+		status, err := c.GetDependenciesStatus(groupID, appID)
```

Comment:
I originally had this but was worried this assumption may not necessarily be always true in case something somehow goes wrong although I suppose even if it did this wouldn't really change the outcome.